parsedown

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env php
+<?php
+
+require(dirname(__FILE__).'/Parsedown.php');
+
+// From a file
+if (isset($argv[1])) {
+
+	if ( ! is_readable($argv[1])) {
+		throw new RuntimeException(sprintf(
+			'Could not read "%s"',
+			$argv[1]
+		));
```

Comment:
```php
<?php
throw new RuntimeException('Could not read "..."');
```
renders as:
```
$ php php.php 
PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Could not read "..."' in /tmp/php.php:3
Stack trace:
#0 {main}
  thrown in /tmp/php.php on line 3
```

which could look much nicer, in this case it does not matter what kind of exception it was and in which line of the script itwas thrown.